operations/restrictions.md

```diff
@@ -369,3 +369,111 @@ Removes restrictions from the service.
 ```javascript
 {}
 ```
+
+## Set restrictions
```

Comment:
How about restriction quotas, do they still apply here? We should mention that if there is any problem with quotas when using restrictions/add this should help. 
What are the implications to switch from using restrictions/add to set, can partners just start calling it? Is it a preferred way of creating restrictions using connector API?